esphome/components/hlw8012/sensor.py

```diff
@@ -2,14 +2,18 @@
 import esphome.config_validation as cv
 from esphome import pins
 from esphome.components import sensor
-from esphome.const import CONF_CHANGE_MODE_EVERY, CONF_CURRENT, \
+from esphome.const import CONF_CHANGE_MODE_EVERY, CONF_INITIAL_MODE, CONF_CURRENT, \
     CONF_CURRENT_RESISTOR, CONF_ID, CONF_POWER, CONF_SEL_PIN, CONF_VOLTAGE, CONF_VOLTAGE_DIVIDER, \
     ICON_FLASH, UNIT_VOLT, UNIT_AMPERE, UNIT_WATT
 
 AUTO_LOAD = ['pulse_counter']
 
 hlw8012_ns = cg.esphome_ns.namespace('hlw8012')
 HLW8012Component = hlw8012_ns.class_('HLW8012Component', cg.PollingComponent)
+INITIAL_MODES = {
+    CONF_CURRENT.upper(): hlw8012_ns.HLW8012_INITIAL_MODE_CURRENT,
```

Comment:
```suggestion
    CONF_CURRENT: hlw8012_ns.HLW8012_INITIAL_MODE_CURRENT,
```

And use `, lower=True` in the one_of below.